AGPs/AGP-1.md

```diff
@@ -0,0 +1,172 @@
+---
+AGP: 1
+Title: The Aragon Governance Proposal Process
+Author: John Light (@john-light)
+Status: Draft
+Track: Meta
+Created: 2018-10-12
+---
+
+# AGP-1: The Aragon Governance Proposal Process
+
+**Version 1.0.5**
+
+## What is an AGP?
+AGP stands for Aragon Governance Proposal. An AGP is a document that details a change to the management, allocation, or use of shared resources owned or directly influenced by the Aragon Network. All AGPs must be consistent with the goals and values put forth in [AGP-0](AGP-0.md) (the Aragon Manifesto) and compliant with the requirements outlined in this document, AGP-1. The AGP author is responsible for building consensus within the community for their AGP and documenting dissenting opinions.
+
+## AGP process rationale
+The purpose of the Aragon Governance Proposal Process ("the AGP process") is to provide a structured process for making changes to the shared resources of the Aragon Network. For these shared resources, governance processes are needed to grant or deny access and approve or reject proposed changes. By creating a fair, lightweight, and transparent AGP process, the AGP-1 authors hope to give ANT holders a meaningful say in the governance of the Aragon Association and the Aragon Network and increase the chances of success for both.
+
+## Proposal workflow
+Parties directly involved in the process are the _AGP author_ (you), the _AGP Editors_, the _Aragon Association (“the Association”)_, and _Aragon Network voters_.
+
+Proposals follow this workflow:
+
+* Stage I: Select AGP Track
+* Stage II: Pre-proposal
+* Stage III: Work In Progress
+* Stage IV: Draft Proposal
+* Stage V: Final Proposal
+* Stage VI: Aragon Network Vote
+
+During Stage III-IV AGP Editors will review proposals for formatting, legibility, and compliance with AGP-1, referring to AGP-1 as the basis for accepting, denying, or requesting modifications to a proposal. The role of AGP Editors is further described in the AGP Editors section.
+
+At a high level, the AGP-1 workflow looks like this:
+
+![](../images/AGP-1/AGP-1_1.png)
+
+### Stage I: Select AGP Track
+Before you spend time working on a proposal, make sure the proposal complies with AGP-1 and has a chance of passing review by the AGP Editors and your peers. Review the AGP tracks and their requirements then select the track that you think is best for your proposal. If your proposal meets the requirements, it has a much greater chance of being accepted by AGP Editors and approved by Aragon Network voters.
+
+There are four tracks that an AGP can be categorized into. Select the one you think is best for your AGP:
+
+* Association: proposals for making changes to the Association
+* Finance: proposals for transferring funds from the Association multisig
```

Comment:
Such as?

Reply:
Payment will be released in multiple trans based out of outcome and that should be via multisig. Eventually, this will make system more robust.

Reply:
Right now the way this would work in practice is:

1. Someone proposes transferring funds from the Association multisig to Project XYZ to develop an Aragon App.
2. The proposal is approved in the ANT vote.
3. The Association would execute the multisig transaction to transfer the funds.

Can you give an example of how your suggestion would change this flow?

Reply:
1.  Someone proposes transferring funds from the Association multisig to Project XYZ to  develop an Aragon App.
   2. The proposal is approved in the ANT vote with multiple checkpoints
   3. The Association would execute the multisig transaction to transfer the funds when each checkpoints got achieved.

Reply:
I see what you mean now, thanks for clarifying. Right now full time teams will be given one year of funding at a time (so they can better plan around hiring and other resource allocation) and Nest teams already have milestone-based funding. Having ANT holders vote on milestone-based funding seems to me premature at this stage but it could evolve to this in the future. I'll leave this suggestion up for others to consider and comment on too :)

Reply:
I think its worth noting that a proposal could be made to fund an organization that operates like that. Whether it is the association or another organization (or multi-sig) that acts as a intermediary between ANT holders and fund recipients.

I also know that the Aragon DAC is very interested making the liquid pledging contracts fully compatible with Aragon Core, those contracts follow a similar principle of milestone based funding with a delegate (or multiple) sitting between the original funder and the funding recipient. So that may provide a really straightforward way for projects to make those types of proposals. 

From the perspective of AGP1, I think it makes sense to not impose the restriction that all proposals are milestone based as I think it reduces flexibility and increases complexity without any significant gain (because milestone based proposals are already possible within the framework, and through the process ANT holders could simply only approve milestone based proposals.)